public/assets/js/scripts.js

```diff
@@ -146,4 +146,97 @@ $(document).ready(function () {
     });
   });
 
+  // use ajax to update select options on index page
+  var state = "#state";
+  $.ajax({
+    type: 'POST',
+    url: './includes/find-house.php',
+    data: {
+      load_state: state
+    },
+    success: function(data){
+      data = JSON.parse(data);
+      var text = "<option selected disabled>Select state</option>";
+
+      for (let i = 0; i < data.length; i++) {
+        text += `<option value="${data[i].state_id}">${data[i].state_name}</option>`;
+      }
+
+      $(state).html(text);
+      $("#area").prop('disabled', true);
```

Comment:
I think it is also preferable to assign the area and type tag to a variable as done  for the state variable above so that they can be used directly like this
```suggestion
      area.prop('disabled', true);
```